src/index.js

```diff
@@ -7,13 +7,7 @@ import util from 'util';
 import JSON5 from 'json5';
 
 function Json5Loader(source) {
-  let value;
-
-  try {
-    value = JSON5.parse(source);
-  } catch (e) {
-    throw new Error('Error parsing JSON5', (e));
-  }
```

Comment:
This was not doing anything except swallowing JSON5's error info. An error is thrown either way, so might as well do nothing and get the useful info.

Reply:
Do you mean switch it from throwing entirely (even though it was not emitting before)?

Reply:
Before loader swallow error, with `this.emitError` it will work as expected. Just:
```
try {	
    value = JSON5.parse(source);	
  } catch (e) {	
    this.emitError(e);	
  }
```

Reply:
Gotcha. Will do when I get home from work.